config/examples/osc/env

```diff
@@ -17,3 +17,6 @@ OOD_BRAND_LINK_ACTIVE_BG_COLOR='#990c23'
 
 OOD_DASHBOARD_DOCS_URL="https://www.osc.edu/ondemand"
 OOD_DASHBOARD_PASSWD_URL="https://my.osc.edu"
+
+# show all apps index page
+SHOW_ALL_APPS_LINK=1
```

Comment:
Should this also be added to `config/examples/awesim/env` since we are keeping AweSim and OSC OnDemand in sync now?

Reply:
Yeah. If we do this, we should probably add shared apps (or featured apps) section at the top of this page... or to the far right after Gateway Apps.